src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -30,7 +30,11 @@ public static int Main(string[] args)
             if (args.Contains("--debug"))
             {
                 args = args.Skip(1).ToArray();
-                Debugger.Launch();
```

Comment:
Debugger.Launch not working looks like a separate bug.  Do runtime folks know about this?